### ISSUE TITLE
Nested INNER JOIN results are nullable

### ISSUE THREAD

**Issue by North101** [simolus3/moor#1195]
```
listStuff:
SELECT
  table1.**,
  table2.**
FROM table1

INNER JOIN table2
  ON something
```

Both `ListStuffResult.table1` and `ListStuffResult.table2` are nullable which isn't possible.

**Comment by simolus3** [comment]
I think that I've fixed this in `moor_generator` 4.3.0 which has been released moments ago, can you try upgrading? If the problem still happens on the latest version I'll take a more detailed look.

**Comment by North101** [comment]
Updated and rebuilt and I have about 76 "The '!' will have no effect" warnings to fix.

Thanks!
